### ISSUE TITLE
Option to override `fetch`

### ISSUE THREAD

**Issue by koenoe** [blakejoy/tmdb-ts#58]
**Is your feature request related to a problem? Please describe.**
When using NextJS we use a patched fetch that enables automatic caching in NextJS. When using a library like cross-fetch, you opt-out of this behaviour.

**Describe the solution you'd like**
Option to pass a fetch method to the TMDB constructor. Basically what graphql-request also offers:.

**Describe alternatives you've considered**
Remove cross-fetch dependency as it shouldn't be needed anymore in Node.

**Comment by blakejoy** [comment]
Good call. What I'll do is bump the major version with the removal of cross fetch and then make the min node version >18.0.0

**Comment by m-ali-akbay** [comment]
The changes fulfills only half of the ticket.

>Describe the solution you'd like
>Option to pass a fetch method to the TMDB constructor. Basically what graphql-request also offers:.

This is still missing.

**Comment by blakejoy** [comment]
Updated in #80 @m-ali-akbay let me know if that works for your use case!
